src/pages/index.js

```diff
@@ -1,4 +1,4 @@
-import Card from "../components/card.js";
+import Card from "../components/Card.js";
 
 import FormValidator from "../components/FormValidator.js";
```

Comment:
Please, delete `this._toggleButtonState();` from line 76 from `FormValidator` because you need to disable the button only in block `then`, otherwise you'll not be able to click it for the 2nd time if there is a server error